### PR TITLE
Update Home Page of Docs

### LINKED ISSUE #167: Update Home Page of Docs

### PR DESCRIPTION
**Pull request by jeff-hernandez**
Closes #167 by updating home page of [the docs](https://compose.alteryx.com/en/update_home_page/index.html).

### CODE REVIEW

File: docs/source/index.rst [hunk 1]
```diff
@@ -1,26 +1,41 @@
...
+
+.. image:: images/workflow.png
+    :align: center
+
+|
+
+By automating the early stage of the machine learning pipeline, our end user can easily define a task and solve it.
 
-**Compose** is advanced software for automating the prediction engineering process. Compose enables you to systematically define prediction problems by automatically extracting historical training examples to train machine learning algorithms.
+.. include:: main_concepts.rst
```
Comment (rwedge) [inline docs/source/index.rst:41]: what if we moved the main_concepts content into this file directly?

Comment (jeff-hernandez) [reply docs/source/index.rst:41]: There's a link in [this guide](https://compose.alteryx.com/en/update_home_page/user_guide/data_slice_generator.html#Data-Slices) that renders the main concepts section specifically. Not sure how to can reference that section specifically if it's included directly.

Comment (rwedge) [reply docs/source/index.rst:41]: Perhaps this would work https://www.sphinx-doc.org/en/1.7/markup/inline.html#role-ref

Comment (jeff-hernandez) [reply docs/source/index.rst:41]: Yeah, that should work. I'll include the file directly.

**Review by rwedge — APPROVED** [review]
Looks good!

**Pull request merged**